jenkins/Jenkinsfile.main

```
@@ -0,0 +1,89 @@
+pipeline {
+  agent none
+
+  options {
+    disableConcurrentBuilds()
+    buildDiscarder(
+      logRotator(
+        artifactDaysToKeepStr: '',
+        artifactNumToKeepStr: '',
+        daysToKeepStr: '',
+        numToKeepStr: '100'))
+    rateLimitBuilds(throttle: [count: 1, durationName: 'hour', userBoost: true])
+  }
+
+  // Daily trigger for daily builds
+  triggers {
+    cron 'H H(20-23) * * *'
+  }
+  
+  parameters {
+    booleanParam name: 'DAILY', description: 'Builds daily builds if true; builds hourly builds if false.'
+    booleanParam defaultValue: true, description: 'Publish the build to S3.', name: 'PUBLISH'
+    booleanParam defaultValue: false, description: 'Force build even if there are no changes.', name: 'FORCE_BUILD'
+    booleanParam defaultValue: false, description: 'Rebuild binaries even if they already exist on dailies page' name: 'FORCE_REBUILD'
```

I think `FORCE_BUILD_DOCKER` might be missing here!

```suggestion
    booleanParam defaultValue: false, description: 'Force build even if there are no changes.', name: 'FORCE_BUILD'
    booleanParam defaultValue: false, description: 'Force rebuild docker images', name: 'FORCE_BUILD_DOCKER'
    booleanParam defaultValue: false, description: 'Rebuild binaries even if they already exist on dailies page' name: 'FORCE_REBUILD'
```